hedera-node/hedera-addressbook-service/src/main/java/com/hedera/node/app/service/addressbook/AddressBookServiceDefinition.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

Comment:
Shouldn't these copyrights be 2024-2025 in all new files?

Reply:
Changed.